canonical-web-and-design/maas-ui
# Add domains details header

## Done

- Added the header to the domains details view complete with the number of hosts/resource records
- Added buttons to show the add and delete forms with dummy content
- Fixed a typo on "Authoriathitavtive" in the column name

## QA
Go to http://0.0.0.0:8400/MAAS/r/domain/0 and check that the header renders correctly

## Fixes

Fixes: #https://github.com/canonical-web-and-design/app-squad/issues/43

![image](https://user-images.githubusercontent.com/11927929/121887393-f5b22700-cd16-11eb-9861-1d3477b4e411.png)
